pkg/supportbundle/collect.go

```diff
@@ -227,14 +227,16 @@ func collectRemoteHost(ctx context.Context, collectSpecs []*troubleshootv1beta2.
 
 		isExcluded, _ := collector.IsExcluded()
 		if isExcluded {
-			opts.ProgressChan <- fmt.Sprintf("[%s] Excluding host collector", collector.Title())
+			msg := fmt.Sprintf("[%s] Excluding host collector", collector.Title())
+			opts.CollectorProgressCallback(opts.ProgressChan, msg)
```

Comment:
what's the reason for not just sending directly to the channel and wrapping it in a callback?

Reply:
It is required for Kotsadm front to pick up the host remote collector progress. I have put the details in the description

Reply:
this matches the pattern used with in-cluster collectors, so that KOTS can display the collector progress.  Previously, KOTS didn't know about host collectors so it didn't matter.